### ISSUE TITLE
Timer not running down after page refresh

### ISSUE THREAD

**Issue by GatoPharaoh** [custom-cards/button-card#349]
**Checklist**
- [x] I updated the card to the latest version available
- [x] I cleared the cache of my browser

**Describe the bug**
When a timer is activated via button card, the timer is running down inside the card. After a page refresh, the timer is stuck and won't update until another page refresh
While a refresh will update the timer, it reamins static and does not run down again
See screenshot below.

**Version of the card**
Version: 3.3.6

**To Reproduce**
This is the configuration I used:
```
 - type: "custom:button-card"
    entity: timer.valve1_time
    name: Zeitstart
    template: panel-two_template
       variables:
          vtim: script.vtim1
           waiting: script.waiting1
```

```  
panel-two_template:
    triggers_update: all
    show_label: true
    show_state: true
    color_type: card
    color: rgb(66, 134, 244)
    icon: mdi:clock-start
    styles:
      card:
        - height: 100px
        - width: 75px
    state:
      - operator: template
        value: >
          [[[ 
            return (states[variables.vtim].state == 'on')
            && (states[variables.waiting].state == 'on')
          ]]]
        color: purple
        icon: 'mdi:clock-alert'
      - value: "active"
        color: rgb(50, 205, 50)
      - value: "idle"
        color: var(--paper-card-background-color)

```

**Screenshots**
Screenshot includes the button card after pressing it as well as two page refreshes
![wffq9MJ0uw](https://user-images.githubusercontent.com/12106737/86514712-da0a8500-be13-11ea-86d4-7467d0068ec5.gif)

**Expected behavior**

**Desktop (please complete the following information):**
 - Browser [Firefox 77.0.1 and Brave 1.10.93]

**Smartphone (please complete the following information):**
 - Device: [iPhone 6S and Galaxy S10]
 - OS:[iOS 13 and Android 10]
 - Browser [Both with latest Home Assistant Official Apps]

**Additional context**
Problem seems to have started after updating to Home Assistant Core 0.111.x
`triggers_update: all` is included and does not fix the problem

**Comment by GatoPharaoh** [comment]
to follow up on the issue
changing any attribute of the timer via `localhost:8123/developer-tools/state` does make the countdown work again until the page is refreshed.

**Comment by RomRider** [comment]
This is fixed in 3.7.0-0 beta release, I'm closing it.
